infrastructure/helm-chart/charts/apps/charts/sources-chatplugin/templates/deployment.yaml

```diff
@@ -52,6 +52,11 @@ spec:
             configMapKeyRef:
               name: kafka-config
               key: KAFKA_COMMIT_INTERVAL_MS
+        - name: CHATPLUGIN_NAME
+          valueFrom:
+            configMapKeyRef:
+              name: sources-chatplugin
+              key: NAME
```

Comment:
Dependency which is not required, but so that the chatplugin-source doesn't start upfront.